sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -247,9 +247,37 @@ export class MessageSender extends LinkEntity {
    * @param sendBatch Boolean indicating whether the encoded message represents a batch of messages or not
    * @return {Promise<Delivery>} Promise<Delivery>
    */
-  private _trySend(encodedMessage: Buffer, sendBatch?: boolean): Promise<void> {
+  private _trySend(
+    encodedMessage: Buffer,
+    sendBatch?: boolean,
+    options?: GetSenderOptions
```

Comment:
Since the sender options doesnt change between multiple invocations to the `send` method and is the same for a given instance of the `MessageSender` object, how about passing the options to the `MessageSender` constructor and have it stored as a property on the instance?

Reply:
Makes sense, added a new private property `_getSenderOptions`